makePoplogCommander.sh

```diff
@@ -231,7 +234,7 @@ static Vector bump( Vector r, int n ) {
 
         //  But we want to grow by a factor to stop repeated linear
         //  extensions becoming O(N^2). We use a factor of 1.5.
-        int delta = ( r-> size ) >> 2;
+        int delta = ( r-> size ) >> 1;
```

Comment:
This was a bug right? It should have been `>> 1` to grow by 1.5?

Reply:
It was a bug. And a stupid one.

Reply:
Is there much point doing the right shift over a division? It'd be clearer with the division. I'm not entirely sure what the difference in semantics are, but they appear to compile to different op codes: https://godbolt.org/z/38xoPhE57